blog/2024-04-19-what-is-the-10101-coordinator.md

```diff
@@ -0,0 +1,81 @@
+
+### Control
+
+When submitting an order the coordinator will look for one or several matching counterparties. Once found the coordinator initiates the dlc protocol with each party. That means your direct counterparty will always be the 10101 coordinator, but implicitely whoever the coordinator matched you up with.
+
+For executing order all parties had to be online to exchange the required messages to establish a dlc. But once this is setup any 10101 Trader can independtly control their position, without having to rely on their indirect counterparty to be online.
+
+#### Simple example
+
+1. Alice wants to go $500 long and Bob wants to go $500 short. Both submit an order to the coordinator.
+1. The coordinator matches Alice with Bob and iniates the dlc protocol.
+1. The coordiantor establishes a dlc with Alice acting as the counterparty going $500 short.
+1. The coordinator establishes a dlc with Bob acting as the counterparty going $500 long.
+
+Assume Bob wants to close his position.
+
+1. Bob submits a closing order going $500 long.
+1. The coordinator will look for a matching order going $500 short.
+1. If a match can be found (doesn't have to be Alice) the coordinator executes the same procedure as before with Alice.
+1. If no match can be found. The order will get rejected or remain open until a match has been found.
+
+Two key take aways from that simple example:
+
+- The coordinator provides a service to the traders to trade whenever they want. Giving them control over their position without impacting the original counterparty.
+- Your position remains open until it either expires, get liquidated or the trader closes the position. If your originally matched counterparty decides to close their position, the coordinator will look for a replacement without you even noticing.
+
+### Speed
+
+The bitcoin price moves fast and so must the trade. In a central exchange a trade is merely a database update. It can't really get faster than that, but with dlc channels we can get very close to it.
+
+At 10101 your first trade will open a dlc channel. A multi-sig transaction that locks up your and your counterparties (coordinators) trading balance on-chain. This first transaction (funding tx) can take some time depending on the fee environment. But once confirmed any consecutive trade is nearly instant.
+
+The hub and spoke design pattern of 10101 allows to optimize the generation of these dlc channels and thus minimizes the corresponding on-chain costs. Assume in the diagram provided above, every 10101 Trader would open a direct dlc channel with each other 10101 Trader. That would result in 30 (n \* n-1 | n=6) channels. With the 10101 coordinator in the middle we only need 6 (n | n=6) channels.
```

Comment:
We should add an additional paragraph about **Capital efficiency** as  in the extreme example the overal capital is spread among 30 channels, while in the 1-coordinator example, all capital can efficiently be used.